StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetGDPRConfirmFlowTests.swift

```diff
@@ -393,6 +395,7 @@ final class PaymentSheet_GDPR_ConfirmFlowTests: STPNetworkStubbingTestCase {
             configuration: configuration,
             authenticationContext: self,
             intent: intent,
+            elementsSession: ._testValue(intent: intent),
```

Comment:
We should be passing in the value that is defined in each test case rather than passing in this test value.